src/components/composite/AlphaBanner/AlphaBanner.styled.tsx

```diff
@@ -0,0 +1,54 @@
+import Typography from '../../atomic/Typography/Typography';
+import { styled } from '@mui/material/styles';
+import { colors } from '@theme';
+import { keyframes } from '@mui/system';
+import Box from '@mui/material/Box';
+
+const slide = keyframes`
+  from { 
+    transform: translateX(0); 
+  }
+  to { 
+    transform: translateX(-50%); 
+  }
+`;
+
+export const BannerContainer = styled(Box)`
+  background: linear-gradient(90deg, #201b35 0%, #372d65 50.52%, #2f2a54 100%);
```

Comment:
I think we had issues with this in the past. It would make the UI load really slow on chrome with 'hardware acceleration' turned off. Let's check if it has the same effect in that context

Reply:
Yes check it please. I think problem was impl. of prev solution 😆